tests/AutoReview/DescribeCommandTest.php

```diff
@@ -1,64 +0,0 @@
-<?php
-
-/*
- * This file is part of PHP CS Fixer.
- *
- * (c) Fabien Potencier <fabien@symfony.com>
- *     Dariusz Rumiński <dariusz.ruminski@gmail.com>
- *
- * This source file is subject to the MIT license that is bundled
- * with this source code in the file LICENSE.
- */
-
-namespace PhpCsFixer\Tests\AutoReview;
-
-use PhpCsFixer\Console\Application;
-use PhpCsFixer\Console\Command\DescribeCommand;
-use PhpCsFixer\FixerFactory;
-use PHPUnit\Framework\TestCase;
-use Symfony\Component\Console\Tester\CommandTester;
-
-/**
- * @internal
- *
- * @coversNothing
- * @group auto-review
- */
-final class DescribeCommandTest extends TestCase
```

Comment:
it's on purpose indeed. checking execution for all real fixers (not mociking one) is not unit test anymore